styles.css

```diff
@@ -0,0 +1,218 @@
+body {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please change the name `styles.css` ❌ to `index.css` ✔️  like others!

![image](https://user-images.githubusercontent.com/63915024/192370875-aef19cd1-c158-4c11-89b7-7af94f22bec1.png)

Reply:
Thank you. I have changed it.